docs/web-wallet.md

```diff
@@ -0,0 +1,249 @@
+
+:::
+
+Below is a code example for integrating the Web Wallet on the client side in JS:
+<Tabs>
+<TabItem value="Off-Chain Verification">
+
+```js
+// Define the verification request
+const verificationRequest = {
+  backUrl: "https://my-app.org/back",
+  finishUrl: "https://my-app.org/finish",
+  logoUrl: "https://my-app.org/logo.png",
+  name: "My app",
+  zkQueries: [
+    {
+      circuitId: "credentialAtomicQuerySigV2",
+      id: 1711399135,
+      query: {
+        allowedIssuers: ["*"],
+        context:
+          "https://raw.githubusercontent.com/anima-protocol/claims-polygonid/main/schemas/json-ld/pol-v1.json-ld",
+        type: "AnimaProofOfLife",
+        credentialSubject: {
+          human: {
+            $eq: true,
+          },
+        },
+      },
+    },
+  ],
+  callbackUrl: "https://my-app.org/api/callback",
+  verifierDid: "did:polygonid:polygon:amoy:2qV9QXdhXXmN5sKjN1YueMjxgRbnJcEGK2kGpvk3cq",
+};
+
+// Encode the verification request to base64
+const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
+
+// Open the Polygon ID Verification Web Wallet with the encoded verification request
+window.open(`https://web-wallet.polygonid.me${base64EncodedVerificationRequest}`);
+```
+
+</TabItem>
+
+ <TabItem value="On-Chain Verification">
+
+```js
+// Define the verification request
+const verificationRequest = {
+  backUrl: "https://my-app.org/back",
+  finishUrl: "https://my-app.org/finish",
+  logoUrl: "https://my-app.org/logo.png",
+  name: "My app",
+  zkQueries: [
+    {
+      circuitId: "credentialAtomicQuerySigV2",
+      id: 1711399135,
+      query: {
+        allowedIssuers: ["*"],
+        context:
+          "https://raw.githubusercontent.com/anima-protocol/claims-polygonid/main/schemas/json-ld/pol-v1.json-ld",
+        type: "AnimaProofOfLife",
+        credentialSubject: {
+          human: {
+            $eq: true,
+          },
+        },
+      },
+    },
+  ],
+  transactionData: {
+    contractAddress: "0x62811c9e1C8b2397767779BC8ff5Ca48869a61Fc",
+    functionName: "submitZKPResponse",
+    chainId: 80002,
+    network: "polygon-amoy",
+  },
+};
+// Encode the verification request to base64
+const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
+
+// Open the Polygon ID Verification Web Wallet with the encoded verification request
+window.open(`https://web-wallet.polygonid.me${base64EncodedVerificationRequest}`);
+```
+
+</TabItem>
+</Tabs>
+
+`backUrl` represents the URL endpoint of your application where the user will be redirected when they click the ‘Back’ button
+
+`finishUrl` represents the URL endpoint of your application where the user will be redirected once the proof has been successfully generated and, they click the ‘Continue’ button
+
+`name` represents the name of your Application.
+
+`zkQueries` represents the query for what you need to authenticate. Input the generated code snippet of the query in Step 2.
```

Comment:
2 observations here:

 * I'd rephrase "represents the query for what you need to authenticate" to "represents the query you request your users to verify"
 * I would emphasize that regardless of the plural in `zkQueries` and this property being a list, currently we only support one query per verification request. This is though prepared for the future support of multiple queries.

Reply:
Ah sorry the box was there already!

<img width="984" alt="image" src="https://github.com/0xPolygonID/docs/assets/15896177/6b86fde9-725b-471c-b38c-cd4620fed731">

I would move it up though, below `zkQueries`

Reply:
Regarding the content of the box, remember, these are not "Authentication Requests" but "Verification Requests", you keep using the word authentication everywhere 😅

Reply:
changed the content and moved the box above